python/daqconf/core/conf_utils.py

```diff
@@ -491,36 +510,29 @@ def generate_boot(apps: list, ers_settings=None, info_svc_uri="file://info_${APP
         }
 
     daq_app_specs = {
-        "daq_application_ups" : {
-            "comment": "Application profile based on a full dbt runtime environment",
-            "env": {
-                "DBT_AREA_ROOT": "getenv",
-                "TRACE_FILE": "getenv:/tmp/trace_buffer_${HOSTNAME}_${USER}",
-            },
-            "cmd": ["CMD_FAC=rest://localhost:${APP_PORT}",
-                    "INFO_SVC=" + info_svc_uri,
-                    "cd ${DBT_AREA_ROOT}",
-                    "source dbt-env.sh",
-                    "dbt-workarea-env",
-                    "cd ${APP_WD}",
-                    "daq_application --name ${APP_NAME} -c ${CMD_FAC} -i ${INFO_SVC}"]
-        },
         "daq_application" : {
-            "comment": "Application profile using  PATH variables (lower start time)",
+            "comment": "Application profile using PATH variables (lower start time)",
+            "image": image,
```

Comment:
Is `image` supposed to be defined for k8s configurations as well?

Reply:
`image` is only used in k8s configuration

Reply:
Ok, this could be fixed by adding a section `daq_application_k8s` in boot.json.